app/src/main/java/com/duycuong/weather/ui/screen/main/sevenday_weather/NextsDayWeatherFragment.java

```diff
@@ -0,0 +1,26 @@
+package com.duycuong.weather.ui.screen.main.sevenday_weather;
+
+import android.location.Location;
```

Comment:
ERROR: Unused import: android.location.Location. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports